torch/_dynamo/variables/builder.py

```diff
@@ -733,6 +733,8 @@ def index_source(key):
                 guards=make_guards(GuardBuilder.ID_MATCH),
             )
         elif trace_rules.lookup(value) is not None:
+            if is_user_defined_allowed(value):
+                self.tx.output.has_user_defined_allowed_in_graph = True
```

Comment:
What does this do?

Reply:
We added this tag every time we are trying to wrap a user defined allowed, so this is needed. But probably I can move this inside of the lookup function.

Reply:
So are you saying this is preexisting code? But if so, I would have expected that there would be a preexisting site you are deleting, but this is only added?

Reply:
Good catch! This is preexisting code. This was original at 
https://github.com/pytorch/pytorch/blob/d261687d5f56ac8148fab2567cf1fa6dd5264def/torch/_dynamo/variables/builder.py#L695-L701

Since I moved the in_graph_function wrapping out of ```is_allowed```, so I should remove them from the old place.